setup.py

```diff
@@ -24,14 +24,8 @@
 here = os.path.abspath(os.path.dirname(__file__))
 __version__ = imp.load_source('bfalg_ndwi.version', 'bfalg_ndwi/version.py').__version__
 
-# get the dependencies and installs
-with open(os.path.join(here, 'requirements.txt'), encoding='utf-8') as f:
-    reqs = f.read().split('\n')
-with open(os.path.join(here, 'requirements-dev.txt'), encoding='utf-8') as f:
-    test_reqs = f.read().split('\n')
-
-install_requires = [x.strip() for x in reqs if 'git+' not in x]
-tests_require = [x.strip() for x in test_reqs if 'git+' not in x]
+install_requires = []
+tests_require = ['nose==1.3.7', 'coverage==4.3.4']
```

Comment:
I don't know if you found the docs for conda or not, but conda can actually run tests as well during a build. It would be great to eventually bake that in to our process too. https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-section

Reply:
The setup file still needs to look like this as I moved the txt's to the docker folder as they are not needed when developing using conda. I did use a test on the shape recipe so it would be possible to add tests to the ndwi recipe in the venicegeo-conda-recipes repo